plugins/modules/aci_interface_policy_spine_profile.py

```diff
@@ -0,0 +1,280 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2017, Bruno Calogero <brunocalogero@hotmail.com>
+# Copyright: (c) 2020, Shreyas Srish <ssrish@cisco.com>
+# Adapted from aci_interface_policy_leaf_profile
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "certified",
+}
+
+DOCUMENTATION = r"""
+---
+module: aci_interface_policy_spine_profile
+short_description: Manage fabric interface policy spine profiles (infra:SpAccPortP)
+description:
+- Manage fabric interface policy spine profiles on Cisco ACI fabrics.
+options:
+  interface_profile:
+    description:
+    - The name of the Fabric access policy spine interface profile.
+    type: str
+    aliases: [ name, spine_interface_profile_name, spine_interface_profile, interface_profile_name ]
+  description:
+    description:
+    - Description for the Fabric access policy spine interface profile.
```

Comment:
The description of the Fabric access policy spine interface profile.

Reply:
Done.